src/Item/hashes.ts

```diff
@@ -20,26 +24,26 @@ const IMAGE_PATH = 'image.png'
 function buildStandardWearableEntityMetadata(
   item: ItemAttributes,
   collection: CollectionAttributes
-): StandardWearableEntityMetadata {
+): StandardWearable & { emoteDataV0?: { loop: boolean } } {
   if (!isStandardItemPublished(item, collection)) {
     throw new Error(
       "The item's collection must be published to build its metadata"
     )
   }
 
-  const entity: StandardWearableEntityMetadata = {
+  const entity: StandardWearable & { emoteDataV0?: { loop: boolean } } = {
     id: getDecentralandItemURN(item, collection.contract_address!),
     name: item.name,
     description: item.description,
     collectionAddress: collection.contract_address!,
-    rarity: item.rarity!,
-    i18n: [{ code: 'en', text: item.name }],
+    rarity: (item.rarity! as unknown) as Rarity,
```

Comment:
I wasn't doing it because Rarity doesn't have a way of returning all of the rarity values without a change in the `common-schemas` lib. I've changed everything to use Rarity except in the Item schema.